source/SIL.AppBuilder.Portal.Frontend/src/data/schema.ts

```diff
@@ -28,6 +28,17 @@ const schemaDefinition: SchemaSettings = {
         users: { type: 'hasMany', model: 'user', inverse: 'organizations' }
       }
     },
+    task: {
+      attributes: {
+        project: { type: 'string' },
```

Comment:
project and product will be relationships

Reply:
(unless this is a temporary thing)

Reply:
Ok will add the schema with just name for each

Reply:
done